ui/components/logs/log-detail-sheet.tsx

```
@@ -134,62 +134,72 @@ export function LogDetailSheet({ log, open, onOpenChange }: LogDetailSheetProps)
+              code={toolsParameter}
+              lang="json"
+              readonly={true}
+              options={{ scrollBeyondLastLine: false, collapsibleBlocks: true, lineNumbers: 'off', alwaysConsumeMouseWheel: false }}
+            />
+          </div>
+        )}
 
-				<div className="mt-4 w-full text-center text-sm font-medium">Conversation History</div>
-				{log.input_history && log.input_history.map((message) => <LogMessageView key={message.content.toString()} message={message} />)}
+        <div className="mt-4 w-full text-center text-sm font-medium">Conversation History</div>
+        {log.input_history && log.input_history.map((message, index) => <LogMessageView key={index} message={message} />)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)